src/components/ProjectDetailPage/ProjectInfo.tsx

```
@@ -39,7 +39,7 @@ export default function ProjectInfo({ projectId }: ProjectInfoProps) {
           {detail ? (
             <div
               className="project-detail-content mt-6 text-base text-black80 leading-relaxed"
-              dangerouslySetInnerHTML={{ __html: detail.contentHtml }}
+              dangerouslySetInnerHTML={{ __html: detail.contents.story_html }}
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)